snuba/clickhouse/formatter/expression.py

```diff
@@ -88,7 +88,15 @@ def visit_column(self, exp: Column) -> str:
             ret_unescaped.append(exp.table_name or "")
             ret.append(".")
             ret_unescaped.append(".")
-        ret.append(escape_identifier(exp.column_name) or "")
+            # If there is a table name and the column name contains a ".",
+            # then we need to escape the column name using alias regex rules
+            # to clearly demarcate the table and columns
+            if "." in exp.column_name:
+                ret.append(escape_alias(exp.column_name) or "")
+            else:
+                ret.append(escape_identifier(exp.column_name) or "")
```

Comment:
Any reason for not just doing this:
```
ret.append(escape_alias(exp.column_name) or "")
```
If there is no '.' in the column name, the result of escaping an identifier or an alias is the same.

Reply:
Done. Initially I thought I might need to change some test cases by using escape_alias even though it won't be necessary. But there aren't additional tests which have a table name and column name.